boxfiles/setup_lb.sh

```diff
@@ -1,3 +1,66 @@
 #!/bin/bash 
-cp /vagrant/boxfiles/nginx.conf /etc/nginx/nginx.conf
+source /vagrant/caasp_env.conf
+
+cat > /etc/nginx/nginx.conf << EOF
+user  nginx;
+worker_processes  auto;
+
+load_module /usr/lib64/nginx/modules/ngx_stream_module.so;
+
+error_log  /var/log/nginx/error.log;
+error_log  /var/log/nginx/error.log  notice;
+error_log  /var/log/nginx/error.log  info;
+
+events {
+    worker_connections  1024;
+    use epoll;
+}
+
+stream {
+    log_format proxy '$remote_addr [$time_local] '
+                     '$protocol $status $bytes_sent $bytes_received '
+                     '$session_time "$upstream_addr"';
+
+    error_log  /var/log/nginx/k8s-masters-lb-error.log;
+    access_log /var/log/nginx/k8s-masters-lb-access.log proxy;
+
+    upstream k8s-masters {
+EOF
+for NUM in $(seq 1 $NMASTERS); do
+    printf "        server caasp4-master-${NUM}:6443 weight=1 max_fails=1;\n" >> /etc/nginx/nginx.conf
+done
+printf "    }\n" >> /etc/nginx/nginx.conf
+printf "    upstream k8s-workers_http {\n" >> /etc/nginx/nginx.conf
+for NUM in $(seq 1 $NWORKERS); do
+    printf "        server caasp4-worker-1:80 weight=1 max_fails=1;\n" >> /etc/nginx/nginx.conf
+done
+printf "    }\n" >> /etc/nginx/nginx.conf
+
+printf "    upstream k8s-workers_https {\n" >> /etc/nginx/nginx.conf
+for NUM in $(seq 1 $NWORKERS); do
+    printf "        server caasp4-worker-1:443 weight=1 max_fails=1;\n" >> /etc/nginx/nginx.conf
+done
+cat >> /etc/nginx/nginx.conf << EOF
+    }
+
```

Comment:
Does it hurt to have multiple workers/masters defined, if they don't actually exist?
I left them all there initially to allow for easier expansion of the cluster (ie: add a new node) without needing to update the LB.